datarobot_batch_scoring/api_response_handlers/pred_api_v10.py

```diff
@@ -66,6 +66,22 @@ def format_data(result, batch, **opts):
             written_fields = out_fields[1:]
             comb = [row[1:] for row in pred]
 
+    if 'reasonCodes' in single_row:
+        num_reason_codes = len(single_row['reasonCodes'])
+        for num in range(1, num_reason_codes + 1):
+            written_fields += [
+                'explanation_{0}_feature'.format(num),
+                'explanation_{0}_strength'.format(num)
+            ]
+        for in_row, out_row in zip(sorted_rows, comb):
```

Comment:
if sorted_rows can be large i'd prefer the izip here but that's just a nitpick

Reply:
Sure.

Reply:
@pprett It seems that there is no easy way to do it in Python2/3 compatible mode. So I'll leave it as-is.

Reply:
@doleks 

`from six.moves import zip`
https://pythonhosted.org/six/#module-six.moves

Reply:
@Axik thank you for it.